src/main/java/edu/indiana/dlib/amppd/controller/WorkflowController.java

```diff
@@ -26,8 +28,19 @@
 	 * @return
 	 */
 	@GetMapping("/workflows")
-	public GalaxyWorkflow[] getWorkflows() {		
-		return galaxyApiService.getWorkflows();
+	public List<Workflow> getWorkflows() {		
+		List<Workflow> workflows = galaxyApiService.getInstance().getWorkflowsClient().getWorkflows();
```

Comment:
@yingfeng-iu can we refactor this and move all galaxy api code and move them to amp services rather than controllers? What do you both think? @vinita84

Reply:
we do need a workflow controller on the amp side to take amp user actions. and retrieve galaxy workflow on the amp side is one such action. in general we shall not put complicated business logic in controller but in service.